lang/dart.md

```diff
@@ -0,0 +1,206 @@
+---
+lang: Dart
+syntax: dart
+title: Dart is Smart Language!
+author: chandan0000
+---
+
+```dart
+void main(){
+    print("hello world");
+}
+```
+## What is Dart?
+- Dart is a dynamic, class-based, object-oriented programming language with closure and lexical scope. Syntactically, it is quite similar to Java, C, and JavaScript. If you know any of these programming languages, you can easily learn the Dart programming language.
+
+- Dart is an open-source programming language which is widely used to develop the mobile application, modern web-applications, desktop application, and the Internet of Things (IoT) using by Flatter framework. It also supports a few advance concepts such as interfaces, mixins, abstract classes, refield generics, and type interface. It is a compiled language and supports two types of compilation techniques. **[Dart official Website: ](https://dart.dev/tutorials)** https://dart.dev/tutorials
```

Comment:
Remove the extra space at the end as it messes with the markdown renderer.
Change `[Dart official Website: ]` to 👇 

```
[Dart official Website:](https://dart.dev/tutorials))
```